Add overview of pages in Managing Jenkins chapter (#4773)
jenkins-infra/jenkins.io: content/doc/book/managing/index.adoc

= Managing Jenkins

Most standard administrative tasks can be performed from the screens
in the *Manage Jenkins* section of the dashboard.
In this chapter, we look at these screens and explain how to use them.

The tiles displayed on the *Manage Jenkins* page are grouped logically.
Here we discuss the pages that are part of the standard installation.

> Some plugins may add pages to this screen.

Reviewer: Don't need the disclaimer?
```suggestion
Plugins may add pages to this screen.
```

The top of the *Manage Jenkins* screen may contain "Monitors"
that alert you when a new version
of the Jenkins software or a security update is available.

> Each monitor includes links to the _changelog_ that describes the new update
> as well as instructions to download and install the update.

Reviewer: Monitors inform and link to other information
```suggestion
Each monitor includes a description of the issue it is reporting and links to additional information about the issue
```

Inline help is available on most *Manage Jenkins* pages:

> 
> * To access the help, select the `?` icon to the right of each field.
> * Click the `?` icon again to hide the help text.
> 
> Inline help is available on most *Manage Jenkins* pages:

Reviewer: Remove duplication
```suggestion
```

* To access the help, select the `?` icon to the right of each field.
* Click the `?` icon again to hide the help text.

[NOTE]
====
The *Manage Jenkins* screens were modernized in 2020
to provide a more attractive user interface for all users
and a much better experience for users on narrow devices such as tablets and phones.
The main changes made were:

* Configuration screens now use HTML `div` tags
rather than the HTML `table` tags that were used in older releases.
* The screens linked from this page are grouped somewhat logically,
whereas older versions presented a long list of tasks in somewhat random order.
* Some configuration fields were moved or added.

For more information about these and other changes that have been implemented, see:

> * link:https://www.jenkins.io/changelog-stable/[LTS Changelog]

Reviewer: Link to the specific changelog
```suggestion
* link:/changelog-stable/#v2.277.1[2.277.1 LTS Changelog]
```

> * link:https://www.jenkins.io/blog/2020/11/10/major-changes-in-weekly-releases/[Jenkins 2.264+: Major changes in the weekly release line]
> * link:https://www.jenkins.io/doc/upgrade-guide/[Jenkins LTS Upgrade Guide]

Reviewer: ```suggestion
* link:/blog/2020/11/10/major-changes-in-weekly-releases/[Jenkins 2.264+: Major changes in the weekly release line]
* link:/doc/upgrade-guide/[Jenkins LTS Upgrade Guide]
```

only when the plugin is installed on your controller.

== Security group

Screens for configuring security features for your Jenkins instance.
See link:/doc/book/security/[Securing Jenkins] for general information
about managing Jenkins security.

link:system-configuration.adoc[Configure Global Security]::
Set configuration parameters that secure your Jenkins instance.

link:/doc/book/using/using-credentials/#adding-new-global-credentials[Manage Credentials]::
Configure the credentials that provide secure access
to third-party sites and applications that interact with Jenkins.

*Configure Credential Providers*::
Configure credential providers and types

link:users.adoc[Manage Users]::
Manage users defined in the Jenkins user database.
This is not used if you use a different security realm such as LDAP or AD.

== Status Information group

link:system-info.adoc[System Information]::
Displays information about the Jenkins environment.

link:/doc/book/system-administration/viewing-logs/[System Log]::
Jenkins log that contains all `java.uil.logging` output related to Jenkins.

*Load Statistics*::
Displays information about resource utilization on you Jenkins instance.

> 
> *Disk usage*::
> Simple disk usage estimation

Reviewer: Not installed by default
```suggestion
```

Reviewer: Need to add the "Cloud statistics" and the "Load statistics" pages

link:about-jenkins.adoc[About Jenkins]::
Provides version and license information for your Jenkins instance.

== Troubleshooting group

*Manage Old Data*::
Remove configuraion information related to plugins that have been removed from the instance.

== Tools and Actions group

Screens for common management tasks
and management tools that enable you to do administrative tasks without using the UI.

*Reload Configuration from Disk*::
Discard all data that is loaded in memory and reload everything from the file system.
This is useful when you modify configuration files directly on disk.

> link:cli-adoc[Jenkins CLI]::

Reviewer: Fix the reference (don't need 'adoc' suffix)
```suggestion
link:cli[Jenkins CLI]::
```

How to use the Jenkins CLI from a shell or script.

link:/doc/book/managing/script-console/[Script Console]::
Execute an Apache Groovy script for administration, troubleshooting, and diagnostics.

> *Prepare for Shutdown*::

Reviewer: The definition term is already made bold by asciidoc, don't need to bold it again
```suggestion
Prepare for Shutdown::
```

Prevents new builds from starting so that the system can be shut down safely.

== Uncategorized group

> Screens for monitoring the Jenkins controller and agents
> and for launching build agents as Docker containers.
> 
> *Monitoring of Jenkins controller*::
> Monitors memory, cpu, http requests and more on the Jenkins controller.
> 
> *Monitoring of Jenkins agents*::
> Monitors builds, the build queue, and Jenkins agents.
> 
> *Docker*::
> Plugin for launching build agents as Docker containers.

Reviewer: Uncategorized is used for those plugins that have not yet declared the category of their page.

```suggestion
Screens for plugins that have not yet declared the category of their page.
```
